README.md

```diff
@@ -231,3 +231,51 @@ let myCivilDate = new CivilDate(2016, 2, 29);
 let newCivilDate = myCivilDate.plus({years: 1, months: 2});
 //results in civil date with value 2017-4-28
 ```
+
+Quick question to @littledan & @rxaviers : Where do you want me to put this sensibly? I'm not sure a file in the repo is the right place, though I'm willing to be told otherwise.
```

Comment:
Heh, I'm not sure if you really want to have this paragraph...

Any way you could rephrase the following in terms of, starting from the beginning, why the design of the proposal is the way it is? E.g., "omit" rather than "remove", "name" rather than "rename".

If this stuff gets too obscure, you can put it in a separate md document in the repo.